### PR TITLE
Updated Documentation

### PR DESCRIPTION
**Pull request by karna12**
Few changes of corrections and fixes here and there!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,34 +92,34 @@ separate Docker container.
...
+- In February, we noticed a systematic replacement of the word "impact" with the word "effect" on one website. This change is very interesting because while "impact" and "effect" have similar meanings, "impact" is a **stronger** word. So, there is an effort being made to **weaken** the language on existing sites. Our question is in part: what tools would we need in order to have this change **flagged** by our tools and presented to the analyst as **potentially interesting**?
 
 ### Sample Data
 
 [`example-data`](./example-data) contains examples of website changes:
 
-- `falsepos-...` files are cases any filter should catch
-- `truepos...` files are cases of changes we care about
+- `false pos-...` files are cases any filter should catch
+- `true pos...` files are cases of changes we care about
```
Comment (Mr0grog) [inline README.md]: These should stay as one word. This is describing the file names here: https://github.com/edgi-govdata-archiving/web-monitoring/tree/master/example-data

File: README.md [hunk 2]
```diff
@@ -93,34 +92,34 @@ separate Docker container.
...
 
 This is a small but illustrative sample. Many more samples will be made
 available as soon as possible.
 
 ### Schema
 
 This describes the schema of the SQL databases shared by the Rails app in
-web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+web-monitoring-DB and the Python processing backend in web-monitoring-processing.
```
Comment (Mr0grog) [inline README.md]: `DB` should actually lower-case here. It’s the name of this GitHub project: https://github.com/edgi-govdata-archiving/web-monitoring-db

(Actually, would you mind making this a link when you lower-case it again? Thanks!)

File: README.md [hunk 3]
```diff
@@ -154,7 +153,7 @@ in addition to the table-specific fields listed below.
     * `diff_length`: Length (in characters) of the diff identified by the above `diff_with_previous_url`.
     * `diff_hash`: SHA 256 hash of the above diff identified by `diff_with_previous_url`.
     * `diff_with_first_url`: URL to diff view in Versionista (comparing with the first recorded version)
-    * `has_content`: Boolean indicating whether Versionista had raw content for this version. If this is true, the version’s `uri` should have a value (and vice-versa).
+    * `has_content`: Boolean indicating whether Versionista had raw content for this version. If this is true, the version’s `Uri` should have a value (and vice-versa).
```
Comment (Mr0grog) [inline README.md]: Same as above: `uri` is actually the name of a field, so it’s important that it’s all lower-case.

File: README.md [hunk 4]
```diff
@@ -93,34 +92,34 @@ separate Docker container.
...
 
 This describes the schema of the SQL databases shared by the Rails app in
-web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+web-monitoring-DB and the Python processing backend in web-monitoring-processing.
 Review the Definition of Terms section above, which corresponds to these tables.
 
 Every table includes:
 
-* uuid: UUID4 unique indentifier
+* uuid: UUID4 unique identifier
```
Comment (Mr0grog) [inline README.md]: `uri` is actually the name of a field, so it’s important that it’s all lower-case.

File: README.md [hunk 5]
```diff
@@ -138,7 +137,7 @@ in addition to the table-specific fields listed below.
 
 * page_uuid: reference to a Page
 * capture_time: when this snapshot of the Page was acquired
-* uri: path to stored (HTML) data; could be a filepath, S3 bucket, etc.
+* Uri: path to stored (HTML) data; could be a file path, S3 bucket, etc.
```
Comment (Mr0grog) [inline README.md]: `uri` is actually the name of a field, so it’s important that it’s all lower-case.

**Comment by karna12** [comment]
Yes, and you're welcome!

**Comment by weatherpattern** [comment]
Hi @karna12 I'm the software project manager for the Web Monitoring project. Thanks so much for your contribution. It is really appreciated. Would you like to me send you an invite to our slack channel? No worries if you don't join, but it would great to keep in touch. Thanks again.

**Comment by karna12** [comment]
@weatherpattern no problem. sure that sounds great, also if any other documentation would like a fix, I'm happy to help, thanks!!

**Comment by weatherpattern** [comment]
Excellent! Invite is sent. Look for the #webmonitor channel. And thanks again for your help with our documentation.

**Comment by karna12** [comment]
Accepted!

File: README.md [hunk 6]
```diff
@@ -3,7 +3,7 @@
 
 This repository is for EDGI [Web Monitoring Project](https://github.com/edgi-govdata-archiving/web-monitoring) documentation and project-wide issue management.
 
-EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently there is a lot of manual labour that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
+EDGI is already monitoring tens of thousands of pages and will eventually be monitoring tens of millions (or even as many as ~1 billion). Currently, there is a lot of manual labor that goes into reviewing all changes, regardless of whether they are meaningful or not. Any system will need to emphasize usability of the UI and efficiency of computational resources.
```
Comment (patcon) [inline README.md]: haha fwiw, this was canadian spelling, as the project docs were started by canadians :)

Comment (karna12) [reply README.md]: Lool. Words do vary but with some, mean the same xD

**Pull request merged**